docs/sources/send-data/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+menuTitle: Send data
+title: Send log data to Loki
+description: Grafana Loki clients
+weight: 600
+---
+# Send log data to Loki
+
+You can use the following clients to send logs to Grafana Loki:
+
+- [Grafana Agent]({{< relref "/docs/agent" >}}) 
+- [Promtail]({{< relref "../clients/promtail" >}})
+-- [Promtail on AWS EC2]({{< relref "./aws/ec2" >}})
+-- [Promtail on AWS ECS]({{< relref "./aws/ecs" >}})
+-- [Promtail on AWS EKS]({{< relref "./aws/eks" >}})
+- [Docker Driver]({{< relref "./docker-driver" >}})
+- [Fluentd]({{< relref "./fluentd" >}})
+- [Fluent Bit]({{< relref "./fluentbit" >}})
+- [Logstash]({{< relref "./logstash" >}})
+- [Lambda Promtail]({{< relref "./lambda-promtail" >}})
+
+There are also a number of third-party clients, for a list see [Unofficial clients](#unofficial-clients).
+
+The [xk6-loki extension](https://github.com/grafana/xk6-loki) permits [load testing Loki]({{< relref "./k6" >}}).
+
+## Picking a client
+
+While all clients can be used simultaneously to cover multiple use cases, which
+client is initially picked to send logs depends on your use case.
+
+### Promtail
+
+Promtail is the client of choice when you're running Kubernetes, as you can
+configure it to automatically scrape logs from pods running on the same node
+that Promtail runs on. Promtail and Prometheus running together in Kubernetes
+enables powerful debugging: if Prometheus and Promtail use the same labels,
+users can use tools like Grafana to switch between metrics and logs based on the
+label set.
+
+Promtail is also the client of choice on bare-metal since it can be configured
+to tail logs from all files given a host path. It is the easiest way to send
+logs to Loki from plain-text files (e.g., things that log to `/var/log/*.log`).
```

Comment:
Google style prefers "for example" over "e.g.".

It might also make sense to reconsider the use of parentheses, the Google Vale plugin says: "Use parentheses judiciously.".

In this case, I read that sentences as the "plain-text files" are "things that log to `/var/log/*.log`" which I don't think is the intention

Reply:
This was copy/pasted from the original file, and I didn't go through and do any cleanup for the text because I wanted to focus on the file moves and not go down the rabbit hole of cleaning up all the issues in the text.